src/shared/canvas/MyData/MyData.tsx

```diff
@@ -82,6 +82,22 @@ const HomeMyData = (props: Props) => {
         [dateField]: makeDatetimePattern({ dateType, date }),
         // type: dataType,
       }),
+    retry: 2,
+    onError: error => {
+      notification.error({
+        message: 'Error loading data from the server',
+        description: isString(error) ? (
+          error
+        ) : isObject(error) ? (
+          <div>
+            <strong>{(error as any)['@type']}</strong>
```

Comment:
the error object is unknown for the front, we don't have a clear definition of the error response type
and for the `optional chaining ?`, it won't do the typing error handling, because the error is already a known type in typescript, so it will raise an error since `@type` and `details` are not in `Error` type